tools/setup/lang.json

```diff
@@ -1,59 +1,44 @@
 {
-    "abap": 27,
-    "ada": 25,
-    "awk": 1,
-    "bash": 7,
-    "c": 49,
-    "c#": 47,
-    "c++": 48,
-    "cobol": 26,
-    "cpp": 48,
-    "csharp": 47,
-    "css": 48,
-    "d": 29,
-    "dart": 28,
-    "delphi": 42,
-    "erlang": 10,
-    "fsharp": 19,
-    "go": 34,
-    "groovy": 13,
-    "haskell": 15,
-    "html": 30,
-    "java": 50,
-    "javascript": 51,
-    "js": 43,
-    "julia": 4,
-    "latex": 40,
-    "lisp": 18,
-    "lua": 22,
-    "mask": 2,
-    "math": 50,
-    "matlab": 33,
-    "mql": 9,
-    "mql4": 9,
-    "objective-c": 35,
-    "objectivec": 35,
-    "objectpascal": 42,
-    "pascal": 42,
-    "perl": 40,
-    "php": 44,
-    "pl": 40,
-    "prolog": 16,
-    "python": 46,
-    "quote": 50,
-    "r": 37,
-    "rb": 39,
-    "ruby": 39,
-    "rust": 8,
-    "sas": 30,
-    "scala": 21,
-    "scheme": 14,
-    "sql": 32,
-    "spoiler": 50,
-    "swift": 41,
-    "tex": 40,
-    "text": 1,
-    "vb.net": 45,
-    "vbnet": 45,
-    "xml": 1
+    "default": {
+        "javascript": 27,
+        "python": 26,
+        "java": 25,
+        "go": 24,
+        "rust": 23,
+        "html": 22,
+        "css": 21,
+        "sql": 20,
+        "bash": 19,
+        "powershell": 18,
+        "c#": 17,
+        "php": 16,
+        "typescript": 15,
+        "c++": 14,
+        "c": 13,
+        "kotlin": 12,
+        "ruby": 11,
+        "asm": 10,
+        "vb.net": 9,
+        "swift": 8,
+        "r": 7,
+        "objective-c": 6,
+        "dart": 5,
+        "scala": 4,
+        "perl": 3,
+        "haskell": 2,
+        "julia": 1
```

Comment:
I'm a little sad that this data set is so small; might be worth posting in `#general` in case anyone knows of a "Data set of languages on GitHub" type thing.

Reply:
btw, this list is only for the languages we want to assign a priority to. The rest of the languages (all of them supported by pygments) still appear in the typeahead, but have a priority value of `0` assigned to them. Do you think we should prioritize more languages?

Reply:
I think if we had a better data set handy, I would want to, but I think this is likely good enough given what's readily available.